sqlglot/optimizer/normalize_identifiers.py

```diff
@@ -48,6 +52,9 @@ def normalize_identifiers(expression, dialect=None):
     Args:
         expression: The expression to transform.
         dialect: The dialect to use in order to decide how to normalize identifiers.
+        store_original_column_identifiers: Whether to store the original column identifiers in
```

Comment:
get rid of this flag, i don't think it's necessary

Reply:
For context, I suggested adding this so that `normalize_identifiers` calls that aren't used in combination with `qualify` + `annotate_types` don't have to carry this additional burden where we store stuff in the `meta`.

So basically only store this metadata if we're using `qualify`, assuming that it's likely going to be coupled with `annotate_types`.

Don't feel strongly about it, just FYI on how it came to be.

Reply:
ok